docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -3,18 +3,18 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Getting Started with Quarkus Security
+= Secure your Quarkus application by using basic authentication
```

Comment:
@michelle-purcell 
I would not mind to keep the original title `Getting Started with Quarkus Security` since immediately below we explain it is about showing how to start fast with Quarkus Security using Basic Authentication. IMHO the title should not be specifically about the Basic Authentication, since the whole doc is about quick intro into Quarkus Security touching on several concepts, not only authentication, but also the authorization